### PR TITLE
Modal window for Cart and fixed burger menu icon to center

### PR DESCRIPTION
**Pull request by Hamuud**
if you have any proposal to modal text, write me

[DEMO LINK](https://fe-feb23-code-hunters.github.io/fe_phone-catalog)

### CODE REVIEW

File: src/Pages/Cart/Cart.tsx [hunk 1]
```diff
@@ -65,11 +68,21 @@ const Cart: React.FC = () => {
...
+              />
             </div>
           </div>
         </div>
       </div>
+      <Modal
+        title="Thank you for purchase"
+        onClose={() => setShowModal(false)}
+        showModal={showModal}
+        description="Thank you for your purchase. Seriously. There’s no such thing as too much small business support. Please enjoy 20% off your next order with code CODEHUNTERS_TOP"
```
Comment (prokivskaaa) [inline src/Pages/Cart/Cart.tsx:84]: let's leave this secondary text:
Please enjoy 20% off your next order with promocode CODEHUNTERS_TOP

Make CODEHUNTERS_TOP bold

**Review by AnyaLyaLya — APPROVED** [review]
Maybe you can fix it 
image

File: src/components/Modal/Modal.module.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+$padding: 10px;
+$icon: 50px;
+
+.modal {
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
```
Comment (prokivskaaa) [inline src/components/Modal/Modal.module.scss:14]: let's use BG-color from UI kit

File: src/Pages/Cart/Cart.tsx [hunk 3]
```diff
@@ -65,11 +68,21 @@ const Cart: React.FC = () => {
...
+                label="Checkout"
+                onClick={() => setShowModal(true)}
+                height="48px"
+              />
             </div>
           </div>
         </div>
       </div>
+      <Modal
+        title="Thank you for purchase"
```
Comment (prokivskaaa) [inline src/Pages/Cart/Cart.tsx:81]: let's change it to: Thank you for your purchase

File: src/components/Modal/Modal.module.scss [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+  &__title {
+    @include desktop-tablet-h4;
+    margin: 0;
+  }
+
+  &__body {
+    @include body-text;
+    padding: $padding;
+    border-top: 1px solid $white-color;
```
Comment (prokivskaaa) [inline src/components/Modal/Modal.module.scss:43]: let's get rid of these borders:

File: src/components/Modal/Modal.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+const Modal: React.FC<Props> = ({
+  title,
+  onClose,
+  showModal,
+  description,
+}) => {
+  const navigate = useNavigate();
+
+  const handleCloseModal = (e: any) => {
```
Comment (prokivskaaa) [inline src/components/Modal/Modal.tsx:25]: Let's change type

**Review by prokivskaaa — APPROVED** [review]

**Pull request merged**